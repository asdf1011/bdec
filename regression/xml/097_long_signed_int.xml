<protocol>
    <sequence name="blah">
        <field name="int32" length="32" type="signed integer" />
        <field name="int64" length="64" type="signed integer" />
    </sequence>
</protocol>
