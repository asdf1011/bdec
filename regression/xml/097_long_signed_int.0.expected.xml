<blah>
    <int32>-15</int32>
    <int64>-10000000000</int64>
</blah>
